packages/react-composites/playwright.config.ts

```diff
@@ -21,6 +26,7 @@ const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
 
   use: {
+    viewport: DESKTOP_VIEWPORT,
```

Comment:
Doesn't appear to be setting the page size judging by the snapshots, investigating

Reply:
Needed to use the playwright launched browser and not spawn our own browser

Reply:
When we do browser.newPage it spawns a new context that doesn't retain the viewport size, seems like a bug, updating playwright version to see if its fixed - and to give access to context in the fixture file that they added so we can use the viewport size in that

Reply:
Yep, updating to 1.15.1 fixed the issue